docs/Options.md

```diff
@@ -45,7 +45,7 @@ Default: `null`
 Defines the capabilities you want to run in your WebDriver session. Check out the [WebDriver Protocol](https://w3c.github.io/webdriver/#capabilities) for more details. If you run an older driver that doesn't support WebDriver you need to apply the [JSONWireProtocol capabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities) to successfuly run a session. Also a useful utility is the Sauce Labs [Automated Test Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/)
 that helps you to create this object by clicking together your desired capabilities.
 
-Type: `Object`<br>
+Type: `Object`|`Object[]`<br>
```

Comment:
This is not quite right .. it should be just `Object` because it describes the webdriverio remote options. It should be moved to `WDIO Options`

Reply:
I am not quite sure I understand. According to the documentation it should be an array of DesiredCapabilities. https://webdriver.io/docs/configurationfile.html
Can you elaborate.

Reply:
`desiredCapabilities` is not an option anymore. The option section you have changed belongs to `webdriverio` package options. Only if you run webdriverio with the wdio testrunner the `capability` should be an array.

Reply:
options for webdriver: https://github.com/webdriverio/webdriverio/blob/master/packages/webdriver/src/constants.js#L3-L92
options for webdriverio (as standalone package): same as webdriver
options for wdio: https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/src/constants.js#L31

Reply:
services option is missing from both ;)
But I guess it belongs to webdriverio, so I will add it there.

Reply:
@christian-bromann And what about `execArgv`, in which one does that belong?

Reply:
@christian-bromann Pushed some changes, hope this is better.